lib/resources/example_json/r4_examples_questionnaire_response.rb

```diff
@@ -14,7 +70,7 @@ module Resources
       'link': [
         {
           'relation': 'self',
-          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/QuestionnaireResponse?_id=SH-12508041'
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/QuestionnaireResponse?patient=SH-12508041'
```

Comment:
i think u kept this as a search by _id in the example url yes?

Reply:
correct: https://github.com/cerner/fhir.cerner.com/pull/669/commits/8e8728f5e996182a9f2e988e9272637f6c60b0cf